### PR TITLE
Feature/SK-707 | Add config support to flower adapter

### PR DESCRIPTION
**Pull request by viktorvaladi**
Adds built-in config support to the flower adapter, for flower clients which use that.

### CODE REVIEW

File: examples/flower-client/client/python_env.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: flower-client
+build_dependencies:
+  - pip
+  - setuptools
+  - wheel==0.37.1
+dependencies:
+  - torch==2.2.1
+  - torchvision==0.17.1
+  - fire==0.3.1
+  - fedn==0.8.0
```
Comment (ahellander) [inline examples/flower-client/client/python_env.yaml:10]: Should be 0.9.0 when we release.

**Review by ahellander — APPROVED** [review]
Looks good. See comment about FEDn-version.

**Comment by Wrede** [comment]
I suggest that you also replace the build scripts with "fedn package create" and "fedn run build".  Maybe remove ENTRTPOINT_OPS since they are confusing.

File: examples/flower-client/README.rst [hunk 2]
```diff
@@ -15,28 +15,32 @@ Running the example
...
+   pip install fedn
 
-Activate the virtual environment:
+Clone this repository, then locate into this directory:
 
 .. code-block::
 
-   source .flower-client/bin/activate
+   git clone https://github.com/scaleoutsystems/fedn.git
+   cd fedn/examples/mnist-pytorch
```
Comment (Wrede) [inline examples/flower-client/README.rst:29]: flower-client

**Review by Wrede — CHANGES_REQUESTED** [review]

**Review by Wrede — APPROVED** [review]

**Pull request merged**